### PR TITLE
Design/fix style

### PR DESCRIPTION
**Pull request by rolfvdhulst**
Apply clang-format to roboteam_world

### CODE REVIEW

File: include/roboteam_world/net/netraw.h [hunk 1]
```diff
@@ -1,49 +1,52 @@
...
+
+    bool operator==(const Address &a) const { return (addr_len == a.addr_len && memcmp(&addr, &a.addr, addr_len) == 0); }
+    void copy(const Address &src) {
+        memcpy(&addr, &src.addr, src.addr_len);
+        addr_len = src.addr_len;
+    }
+    void reset() {
+        memset(&addr, 0, sizeof(addr));
+        addr_len = 0;
+    }
```
Comment (Yuhanun) [inline include/roboteam_world/net/netraw.h]: What... is this C?

**Comment by Yuhanun** [comment]
Only formatting changed?

**Comment by rolfvdhulst** [comment]
> Only formatting changed?

Yeah, only applied clang-format

File: include/roboteam_world/net/netraw.h (hunk 1 above)
Comment (rolfvdhulst) [reply include/roboteam_world/net/netraw.h]: Basically. It's the library provided by SSL to listen to protobuf messages, so it's basically the best guarantee we have at receiving protobuf. It's the same as SSL-Vision uses.

**Pull request merged**